src/USER-OMP/reaxc_forces_omp.cpp

```diff
@@ -315,9 +316,10 @@ void Validate_ListsOMP( reax_system *system, storage * /*workspace */, reax_list
         else comp = hbonds->num_intrs;
 
         if (End_Index(Hindex, hbonds) > comp) {
-          fprintf(stderr,"step%d-hbondchk failed: H=%d end(H)=%d str(H+1)=%d\n",
+          char errmsg[256];
+          snprintf(errmsg, 256, "step%d-hbondchk failed: H=%d end(H)=%d str(H+1)=%d\n",
                   step, Hindex, End_Index(Hindex,hbonds), comp );
-          MPI_Abort( comm, INSUFFICIENT_MEMORY );
+          system->error_ptr->all(FLERR, errmsg);
```

Comment:
Same issue here. This looks more likely for calling `error->one()` instead of `error->all()`.